cmake/onnxruntime_providers.cmake

```diff
@@ -650,6 +650,17 @@ if (onnxruntime_USE_OPENVINO)
           ARCHIVE  DESTINATION ${CMAKE_INSTALL_LIBDIR}
           LIBRARY  DESTINATION ${CMAKE_INSTALL_LIBDIR}
           RUNTIME  DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+  if(NOT WIN32)
+    if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/onnxruntime/capi/)
```

Comment:
would it be better to check whether python bindings are enabled?

Reply:
Hi @jywu-msft . I have reverted my changes. I found the perfect fix for this issue. please, have a look at the changes now.